manifests/init.pp

```diff
@@ -1,6 +1,7 @@
 # == Class: nodejs: See README.md for documentation.
 class nodejs (
   $cmd_exe_path                                        = $nodejs::params::cmd_exe_path,
+  Boolean $manage_nodejs_package                       = $nodejs::params::manage_nodejs_package,
```

Comment:
`$nodejs::params::manage_nodejs_package` is only used in this singe place. Please directly assign the  value in init.pp and don't add another variable to params.pp. That makes it later on easier to migrate to puppet-strings

Reply:
@bastelfreak Done!